src/Lykke.AlgoStore.CSharp.AlgoTemplate.Services/Services/AlgoSettingsService.cs

```diff
@@ -83,12 +83,12 @@ public string GetAlgoInstanceWalletId()
 
         private string GetAlgoInstanceTradedAsset()
```

Comment:
Isn't it better to change methods names to GetAlgoInstanceTradedAssetId and GetAlgoInstanceAssetPairId